components/brave_wallet_ui/components/extension/connect-hardware-wallet-panel/index.tsx

```diff
@@ -41,8 +44,11 @@ function ConnectHardwareWalletPanel (props: Props) {
       return getLocale('braveWalletConnectHardwarePanelConfirmation').replace('$1', walletName)
     }
 
-    if (hardwareWalletCode === 'openEthereumApp') {
-      return getLocale('braveWalletConnectHardwarePanelOpenApp').replace('$1', walletName)
+    if (hardwareWalletCode === 'openLedgerApp') {
+      let network = (coinType === BraveWallet.CoinType.SOL) ? 'Solana' : 'Ethereum'
+      return getLocale('braveWalletConnectHardwarePanelOpenApp')
+        .replace('$1', network)
+        .replace('$2', walletName)
```

Comment:
I added coinType as a prop to ConnectHardwareWalletPanel in order to fix the messaging on the panel that tells the user to open the Solana app on the ledger.  It was currently hardcoded to Ethereum:

> Hardware wallet requires Ethereum App opened on Ledger

By passing coinType we can make this string dynamic and reflect the network of the account.